### DISCUSSION TITLE
我在Hexo中使用了Waline评论插件，主题使用的是Fluid。但每次从本地重新上传部署Hexo时，之前的评论会被全部清空

### DISCUSSION THREAD

**Discussion by mrc321** [walinejs/waline#952] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/46882196/164866263-faba51ab-9d94-4be9-beaf-638c66e5fa9b.png)
上面是fluid主题内对Waline的一些配置，我只是填了一个serverURL

**Comment by lizheming** [comment, marked as answer]
可以进后台看一下评论在不在，前台有可能因为你 URL 格式发生变化了导致评论不展示了。

**Reply by mrc321** [reply]
谢谢，确实是url的问题，因为没有给那篇文章指定url导致是每次重新部署都会生成一个新的url
